docs/whats_new/v0.6.2.md

```diff
@@ -0,0 +1,40 @@
+# Version 0.6.2
+
+v0.6.2 is a security and bug fix release.
+
+This release is backward compatible to 0.6.0 and 0.6.1. As always, only load previously trained models using the same version of AutoGluon that they were originally trained on.
+Loading models trained in different versions of AutoGluon is not supported.
+
+See the full commit change-log here: https://github.com/autogluon/autogluon/compare/v0.6.1...v0.6.2
+
+This version supports Python versions 3.7 to 3.9. 0.6.x are the last releases that will support Python 3.7.
```

Comment:
Add special thanks to first time contributors

Reply:
Added @daikikatsuragawa and @yzhliu ti special thanks